kolibri/plugins/user/assets/src/views/sign-up-page/index.vue

```diff
@@ -136,22 +135,26 @@
         return this.errorCode === 400;
       },
       allFieldsPopulated() {
-        return !(this.name && this.username && this.password && this.confirmed_password);
-      },
-      canSubmit() {
-        return !this.termsAgreement || this.allFieldsPopulated || !this.passwordsMatch || this.busy;
+        return this.name && this.username && this.password && this.confirmed_password;
       },
       errorMessage() {
         return this.backendErrorMessage || this.$tr('genericError');
       },
     },
     methods: {
       signUp() {
-        this.signUpAction({
-          full_name: this.name,
-          username: this.username,
-          password: this.password,
-        });
+        const canSubmit =
```

Comment:
It actually used to be https://github.com/fle-internal/kolibri-instant-schools-plugin/pull/24/files

Changed it due to verbal suggestion from @indirectlylit a while back, iirc. But it's not used anywhere, and it doesn't need `this`, so I'm not sure what the purpose would be